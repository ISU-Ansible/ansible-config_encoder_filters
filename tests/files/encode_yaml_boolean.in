---

encode_yaml_boolena_var1: yes
encode_yaml_boolena_var2: no
encode_yaml_boolena_var3: true
encode_yaml_boolena_var4: false
encode_yaml_boolena_var5: True
encode_yaml_boolena_var6: False
encode_yaml_boolena_var7: "yes"
encode_yaml_boolena_var8: "no"
encode_yaml_boolena_var9: "true"
encode_yaml_boolena_var10: "false"
encode_yaml_boolena_var11: "True"
encode_yaml_boolena_var12: "False"
