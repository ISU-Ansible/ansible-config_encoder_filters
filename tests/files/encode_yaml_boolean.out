encode_yaml_boolena_var1: true
encode_yaml_boolena_var10: "false"
encode_yaml_boolena_var11: "True"
encode_yaml_boolena_var12: "False"
encode_yaml_boolena_var2: false
encode_yaml_boolena_var3: true
encode_yaml_boolena_var4: false
encode_yaml_boolena_var5: true
encode_yaml_boolena_var6: false
encode_yaml_boolena_var7: "yes"
encode_yaml_boolena_var8: "no"
encode_yaml_boolena_var9: "true"
